tox.ini

```diff
@@ -0,0 +1,82 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
```

Comment:
I think we should outsource testing dependencies to a file like `test-requirements.txt`

Reply:
Done